dora/underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -749,6 +749,7 @@ public void setOwner(String path, String user, String group) throws IOException
   @Override
   public void setAttribute(String path, String name, byte[] value) throws IOException {
     if (StringUtils.isEmpty(name)) {
+      LOG.warn("Try to set Xattr to an empty file name.");
```

Comment:
actually is the `name` here file name or user name?
```suggestion
      LOG.warn("Try to set Xattr to an empty file name for path {}", path);
```